.env.example

```diff
@@ -4,8 +4,10 @@ APP_KEY=n96M1TPG821EdN4mMIjnGKxGytx9W2UJ
 NODE_ENV=development
 CACHE_VIEWS=false
 SESSION_DRIVER=cookie
-DB_CONNECTION=sqlite
+SESSION_DOMAIN=example.com
+DB_CONNECTION=mysql
```

Comment:
the reason i changed because most of ppl use mysql and to also keep the db env keys unless the user wants to use sqlite then he can remove them